encord/constants/model.py

```diff
@@ -19,6 +19,15 @@
 from typing import Set, cast
 
 
+class Devices(Enum):
```

Comment:
I think the industry standard is to use the singular for enums. That way you can then use it as `Device.CPU` which makes more sense I think. Can we change that?

Reply:
Good point, renamed to `Device`